presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaRecordSet.java

```diff
@@ -55,11 +54,11 @@
 {
     private static final Logger log = Logger.get(KafkaRecordSet.class);
 
-    private static final int KAFKA_READ_BUFFER_SIZE = 100_000;
     private static final byte[] EMPTY_BYTE_ARRAY = new byte[0];
+    private static final int POLL_TIMEOUT = 1_000;
```

Comment:
I still feel 1s is too long, how about 500ms?

Reply:
Fixed.